gnomad_qc/v5/data_ingestion/create_vds.py

```diff
@@ -0,0 +1,70 @@
+"""Script to combine GVCFs into a single VDS."""
+
+import argparse
+import logging
+
+import hail as hl
+from gnomad.utils.file_utils import create_vds
+
+from gnomad_qc.v5.resources.basics import dragen_tgp_vds
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("create_vds")
+logger.setLevel(logging.INFO)
+
+
+def main(args):
+    """Create VDS from text file of GVCFs paths and write to output path."""
+    hl.init(
+        log="/tmp/gvcf_combiner.log", default_reference="GRCh38", tmp_dir=args.temp_path
+    )
+    gvcfs = args.gvcfs
+    temp_path = args.temp_path
+    save_path = temp_path + "combiner_plan.json"
+    output_path = args.output_path if args.output_path else dragen_tgp_vds.path
```

Comment:
could add logger here to print that dragen_tgp_vds.path is being used if no output path